pset-iterators.js

```diff
@@ -161,6 +206,12 @@ console.log('Problem 6:')
 
 // Add your code below this line
 
+const firstRepublican = presidents.find((freedomFighter)=>{
+  return freedomFighter.president;
```

Comment:
Hmm, this is close but missing something. The .find iteration method returns the first value that meets the condition; that is, your return must satisfy a true/false value. [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L180) is an example of that.